### PR TITLE
Tests Fixtures Update

### PR DESCRIPTION
**Pull request by Averynder**
This PR does 2 things,

1. it moves the user_account permissions which were originally separated from the rest of the code in the django_server into the user_account directory / keeping all the user account code together
2. it replaces the created database entries for each test by loading the entries in the fixtures instead. This lead to a few tests being removed as they were included within another test after simplifying

### CODE REVIEW

**Review by Amanda-Kolopa — COMMENTED** [review]
Everything appears to be good and running on my end.
I'll let @RobertBeaudenon have the final review before merging to double verify.
Perhaps you can fix the detected code smell in the meantime?

File: server/user_account/tests.py
Comment (NicholasNagy) [inline server/user_account/tests.py:229]: Why are you commenting this out?

**Review by NicholasNagy — COMMENTED** [review]
Looks good, thank you for moving things around and cleaning up the tests.

I left a single comment for discussion, before I can make a judgement to merge or not

File: server/user_account/tests.py
Comment (Averynder) [reply server/user_account/tests.py]: sorry, it was a forgotten line to be removed. test_user was completely removed as it had no real purpose and it's attributes were the same as admin. Before deleting code I usually comment it out to verify nothing else is being used or if I want to go back to this code for whatever reason. Good catch!

Comment (NicholasNagy) [reply server/user_account/tests.py]: np!

**Review by NicholasNagy — APPROVED** [review]
Everything looks good now, I will merge it!

**Pull request merged**
